### PR TITLE
:bug: update enablemetrics description

### PR DESCRIPTION
**Pull request by ldpliu**
## Related issue(s)
https://issues.redhat.com/browse/ACM-11627
Fixes #

### CODE REVIEW

**Comment by ldpliu** [comment]
/cc @jc-berger Please help to review the message below. user will see it when they create the global hub.
image

`EnableMetrics enables the metrics for the global hub created kafka and postgres components. For user provided kafka and postgres, it is useless`

**Comment by clyang82** [comment]
@jc-berger Could you help to review the description? Thanks.

File: operator/apis/v1alpha4/multiclusterglobalhub_types.go [hunk 1]
```diff
@@ -96,6 +96,7 @@ type MulticlusterGlobalHubSpec struct {
 	// +optional
 	AdvancedConfig *AdvancedConfig `json:"advanced,omitempty"`
 	// EnableMetrics enables the metrics for the global hub created kafka and postgres components.
+	// For user provided kafka and postgres, it is useless
```
Comment (jc-berger) [inline operator/apis/v1alpha4/multiclusterglobalhub_types.go:99]:
```suggestion
	// For users provided with kafka and postgres, this metric is useless. 
```

File: operator/bundle/manifests/operator.open-cluster-management.io_multiclusterglobalhubs.yaml [hunk 2]
```diff
@@ -284,7 +284,8 @@ spec:
               enableMetrics:
                 default: true
                 description: EnableMetrics enables the metrics for the global hub
-                  created kafka and postgres components.
+                  created kafka and postgres components. For user provided kafka and
```
Comment (jc-berger) [inline operator/bundle/manifests/operator.open-cluster-management.io_multiclusterglobalhubs.yaml:287]:
```suggestion
                For users provided with kafka and postgres, this metric is useless. 
```

File: operator/config/crd/bases/operator.open-cluster-management.io_multiclusterglobalhubs.yaml [hunk 3]
```diff
@@ -284,7 +284,8 @@ spec:
               enableMetrics:
                 default: true
                 description: EnableMetrics enables the metrics for the global hub
-                  created kafka and postgres components.
+                  created kafka and postgres components. For user provided kafka and
```
Comment (jc-berger) [inline operator/config/crd/bases/operator.open-cluster-management.io_multiclusterglobalhubs.yaml:287]:
```suggestion
                For users provided with kafka and postgres, this metric is useless. 
```

**Review by clyang82 — APPROVED** [review]
LGTM

**Comment by clyang82** [comment]
the changes won't be impact on ut and e2e. the errors are not related with this PR. it will be handled separately.
/override ci/prow/test-e2e

**Pull request merged**